functions/api/helpers.js

```diff
@@ -59,19 +59,18 @@ export async function updateKPIProgressionValue(kpiRef, date, value) {
 
   // TODO: Also populate `createdBy`/`editedBy` when using the API. This
   // might be possible by checking the `X-Apigateway-Api-Userinfo` header.
-  let data = {
-    value,
+  data = {
+    ...data,
     timestamp: setHours(date, 12),
     created: new Date(),
     createdBy: null,
   };
 
   if (!valuesSnapshot.empty) {
-    const { comment, created, createdBy } = valuesSnapshot.docs[0].data();
+    const { created, createdBy } = valuesSnapshot.docs[0].data();
 
     data = {
       ...data,
-      comment: comment || null,
```

Comment:
Dette ble opprinnelig lagt til fordi vi fikk tilbakemelding på at man mistet kommentarer lagt inn manuelt ved senere oppdatering av verdiene. Nå tenker vi det blir et krav at kommentaren må postes på nytt? Det gir vel i og for seg kanskje mening, og nå er det jo mulig via API-et. Men kan vi se for oss at noen fortsatt kan komme til å legge inn en manuell kommentar, som så vil forsvinne dersom man har en løsning som f.eks. pusher verdier to ganger i døgnet?

Reply:
Ja, for PUT-kallet er det vel det eneste som gir mening? Vi kunne vel lagt til støtte for PATCH i tillegg da kanskje?